src/openforms/forms/validators.py

```diff
@@ -44,3 +46,23 @@ def validate_not_deleted(form):
             _("Form is deleted."),
             code="invalid",
         )
+
+
+def validate_form_definition_is_reusable(instance: FormDefinition, **kwargs) -> None:
+    """
+    - Ensure that ``is_reusable`` can only be switched from true to false
+      if the form definition is used in less than 2 formsteps.
+    """
```

Comment:
```suggestion
    """
    Validate the integrity of the ``is_reusable`` flag.
    
    ``is_reusable`` can only be switched from true to false if the form definition is used in less than
    two distinct forms.
    """
```
`used_in` returns forms, not formsteps. In practice, this should be the same, but let's use correct wording.